### PR TITLE
#171648165 Setup Coveralls file

### PR DESCRIPTION
**Pull request by YankyJayChris**
#### Description
This PR is for setting up Coveralls code coverage service

#### Type of change

Please select the relevant option

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Others (cosmetics, styling, improvements)
- [ ] This change requires a documentation update

#### How Has This Been Tested?

- [ ] Unit
- [ ] Integration
- [ ] End-to-end
- [x] No testing required

#### How to Test

N/A

#### Any background context you want to add (Optional)

N/A

#### STORY ID

[Finishes #171648165]

### CODE REVIEW

**Review by brianineza01 — APPROVED** [review]

**Review by Agnes-N — APPROVED** [review]

**Review by Irutingabo — APPROVED** [review]

**Review by deschantkn — CHANGES_REQUESTED** [review]
This will not work as we are not using travis pro. That is for paid private repos.
This file is also not necessary as we will use Travis to report coverage. What you need to do is the following:

1. Set up a coverage script in our `package.json` file.
2. Coveralls gives us a coverage badge, you need to add that badge to our `README.md` file

Once that is done @Agnes-N will integrate your coverage command into our `travis.yml` config file since she is the one working on that.

You can reach out if you need more clarification.

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
```
Comment (deschantkn) [inline .coveralls.yml]: We are not using travis-pro, kindly remove this as it has no effect. Unless you have a reason for it being here, if yes let me know.

**Review by deschantkn — CHANGES_REQUESTED** [review]
Great job @YankyJayChris ,
The last step is to connect the coverage script to Travis, so coverage can be reported automatically when builds run on Travis. You can do that by editing the `.travis.yml` file and add this:

```
after_success:
  - npm run coveralls
```

This will ensure that coverage is only reported if tests pass successfully.

**Review by deschantkn — APPROVED** [review]
Looks good

**Pull request merged**